### ISSUE TITLE
Support Array.CreateInstance(...)

### ISSUE THREAD

**Issue by TriEdgeAI** [bridgedotnet/Bridge#2052]
[Array.CreateInstance Method](https://msdn.microsoft.com/en-us/library/system.array.createinstance(v=vs.110).aspx)
> Initializes a new instance of the Array class.

 - [Array.CreateInstance Method (Type, Int32)](https://msdn.microsoft.com/en-us/library/zb3cfh7k(v=vs.110).aspx)
> Creates a one-dimensional Array of the specified Type and length, with zero-based indexing.
```cs
public static Array CreateInstance(elementType, int length)
```

 - [Array.CreateInstance Method (Type, Int32, Int32)](https://msdn.microsoft.com/en-us/library/4hy8w4fs(v=vs.110).aspx)
> Creates a two-dimensional Array of the specified Type and dimension lengths, with zero-based indexing.
```cs
public static Array CreateInstance(elementType, int length1, int length2)
```

 - [Array.CreateInstance Method (Type, Int32, Int32, Int32)](https://msdn.microsoft.com/en-us/library/bybh3298(v=vs.110).aspx)
> Creates a three-dimensional Array of the specified Type and dimension lengths, with zero-based indexing.
```cs
public static Array CreateInstance(Type elementType, int length1, int length2, int length3)
```

 - [Array.CreateInstance Method (Type, Int32[])](https://msdn.microsoft.com/en-us/library/dfs8044k(v=vs.110).aspx)
> Creates a multidimensional Array of the specified Type and dimension lengths, with zero-based indexing. The dimension lengths are specified in an array of 32-bit integers.
```cs
public static Array CreateInstance(Type elementType, params int[] lengths)
```

**Comment by LeonidVeriga** [comment]
Hi @TriEdgeAI,

There is `JSON.ParseAsArray<>` #1438 that can be used as a workaround.

**Comment by TriEdgeAI** [comment]
Oh, no worries. We're not blocked, we figure out the workarounds, I just like to write an issue for every detail I notice. I hope it's helpful :)

**Comment by LeonidVeriga** [comment]
@TriEdgeAI, yes, thank you. We also support this approach to create detailed issues.

**Comment by geoffreymcgill** [comment]
This feature has been implemented and a pull request has been submitted. The feature is expected to be included in the Bridge 15.7 release.
